flyteplugins/go/tasks/pluginmachinery/internal/webapi/launcher.go

```diff
@@ -38,6 +38,8 @@ func launch(ctx context.Context, p webapi.AsyncPlugin, tCtx core.TaskExecutionCo
 	// Store the created resource name, and update our state.
 	state.ResourceMeta = rMeta
 	state.Phase = PhaseResourcesCreated
+	state.PhaseVersion = 2
```

Comment:
It was 2. https://github.com/flyteorg/flyte/blob/e8588f3a04995a420559327e78c3f95fbf64dc01/flyteplugins/go/tasks/pluginmachinery/internal/webapi/launcher.go#L52

because we set it to 0 when we allocate a token. https://github.com/flyteorg/flyte/blob/e8588f3a04995a420559327e78c3f95fbf64dc01/flyteplugins/go/tasks/pluginmachinery/internal/webapi/allocation_token.go#L54